services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -44,19 +49,31 @@ const confirmPublicKey = async publicKey => {
 };
 
 const getPublicKeyByAddress = async address => {
-	const accountsDB = await knex('accounts');
+	const accountsDB = await getAccountsIndex();
 	const [{ publicKey }] = await accountsDB.find({ address });
 	return publicKey;
 };
 
 const getIndexedAccountByPublicKey = async publicKey => {
-	const accountsDB = await knex('accounts');
+	const accountsDB = await getAccountsIndex();
 	const account = await accountsDB.find({ publicKey });
 	return account;
 };
 
 const getIndexedAccountInfo = async params => {
-	const accountsDB = await knex('accounts');
+	const accountsDB = await getAccountsIndex();
+	const account = await accountsDB.find(params);
```

Comment:
Please consider unwrapping `account` at source and returning only the object. This change would also require you to update at places wherever the method has been invoked.

Same could be said for `getIndexedAccountByPublicKey`. But if possible, would appreciate replacing it completely with `getIndexedAccountInfo`. Seems a little redundant. Alternatively, remove `getIndexedAccountInfo` and replace it couple more new methods: `getIndexedAccountByAddress` and `getIndexedAccountByUsername`.

```suggestion
	const [account] = await accountsDB.find(params);
```

Reply:
I removed `getIndexedAccountByPublicKey` and replaced it with updated `getIndexedAccountInfo`.